src/preferences/dialog/dlgpreferences.cpp

```diff
@@ -119,19 +119,7 @@ DlgPreferences::DlgPreferences(
                           new QTreeWidgetItem(contentsTreeWidget, QTreeWidgetItem::Type)),
             tr("Vinyl Control"),
             "ic_preferences_vinyl.svg");
-#else
-    addPageWidget(PreferencesPage(
-<<<<<<< HEAD
```

Comment:
Hmm, are you using the pre-commit hooks? They should catch that and abort committing. If they don't, we need to fix that.

Reply:
I use `--no-verify` with merges. Otherwise they get filled with extraneous noise.

Reply:
I suggest to use `SKIP=clang-format git commit` instead. Then you don't lose all the other hooks.